packages/oidc-middleware/package.json

```diff
@@ -19,7 +19,7 @@
     "test": "yarn test:unit && yarn test:e2e && yarn test:integration",
     "pretest:e2e": "../../scripts/updateSeDrivers.sh",
     "test:e2e": "protractor test/e2e/protractor.conf.js",
-    "test:integration": "../../scripts/tck.sh",
+    "test:integration": "../../scripts/tck.sh 0.4.0-SNAPSHOT",
```

Comment:
Still using older version of TCK in oidc-middleware. Tests failing with latest version. 
Created this JIRA to track and fix it - https://oktainc.atlassian.net/browse/OKTA-234895